docs/basics/userguide/save_load.md

```diff
@@ -149,9 +149,130 @@ Success!
+[[0.1830874  0.15802259 0.12087594 0.16358512]
+ [0.         0.         0.         0.16674384]
+ [0.10049071 0.09517673 0.11326436 0.07236883]
+ [0.10552007 0.06660356 0.07020302 0.1104349 ]
+ [0.23065463 0.         0.19028937 0.20689438]
+ [0.         0.         0.         0.        ]
+ [0.08408004 0.         0.         0.        ]
+ [0.         0.         0.         0.        ]
+ [0.         0.17382279 0.         0.2417443 ]
+ [0.         0.29498867 0.         0.        ]
+ [0.46487572 0.         0.         0.        ]
+ [0.         0.         0.         0.        ]
+ [0.         0.         0.         0.        ]
+ [0.         0.52043878 0.56986596 0.19259904]
+ [0.57831479 0.6858159  0.22998936 0.39350426]
+ [0.         0.         0.         0.        ]]
+```
+
+### Load the agent
+
+To load an agent, you should use its `load()` function. But be careful, you have to add some not-saved parameters (in this case, the environment). This parameters should be given through a `dict`.
+
+
+```python
+# create a seeded env
+env_for_loader = gym_make(env_id)
+env_for_loader.seed(int(seeder.rng.integers(env_seed_max_value)))
+
+# create the 'not-saved parameters' dict
+params_for_loader = dict(env=env_for_loader)
+
+# load the agent
+loaded_agent = QLAgent.load("./results/", **params_for_loader)
+print(loaded_agent.Q)  # print the content of the Q-table
+```
+
+
+```none
+[[0.1830874  0.15802259 0.12087594 0.16358512]
+ [0.         0.         0.         0.16674384]
+ [0.10049071 0.09517673 0.11326436 0.07236883]
+ [0.10552007 0.06660356 0.07020302 0.1104349 ]
+ [0.23065463 0.         0.19028937 0.20689438]
+ [0.         0.         0.         0.        ]
+ [0.08408004 0.         0.         0.        ]
+ [0.         0.         0.         0.        ]
+ [0.         0.17382279 0.         0.2417443 ]
+ [0.         0.29498867 0.         0.        ]
+ [0.46487572 0.         0.         0.        ]
+ [0.         0.         0.         0.        ]
+ [0.         0.         0.         0.        ]
+ [0.         0.52043878 0.56986596 0.19259904]
+ [0.57831479 0.6858159  0.22998936 0.39350426]
+ [0.         0.         0.         0.        ]]
+```
+
+As you can see, we haven't re-fit the agent, and the q-table is the same as the one previously saved.
+
+Then, you can use your agent to solve a test environment :
+
+```python
+# create a seeded test env
+test_env = gym_make(env_id)
+test_env.seed(int(seeder.rng.integers(env_seed_max_value)))
+
+observation, info = test_env.reset()
+for tt in range(50):
+    action = loaded_agent.policy(observation)
+    next_observation, reward, terminated, truncated, info = test_env.step(action)
+    done = terminated or truncated
+    if done:
+        if reward == 1:
+            print("Success!")
+            break
+        else:
+            print("Fail! Retry!")
+            next_observation, info = test_env.reset()
+    observation = next_observation
+```
+
```

Comment:
This could be merged into the previous code to make it shorter and just say "this code show that the loaded agent contains all the necessary component needed to reuse the agent" or something like that.

Reply:
done !